### ISSUE TITLE
Can the number of lines of code be counted during scanning?

### ISSUE THREAD

**Issue by larkzhang** [coffeeandsecurity/DakshSCRA#4]
Can the number of lines of code be counted during scanning? I saw that some statistical information was printed in the log during scanning, such as the number of files and rules. I would like to ask if the number of lines of code can be counted.
The following is the statistical information of log printing
[ [-] Filetypes Selected: ['*.java', '*.jsp']
      [-] Identifying total files to be scanned!
      [-] Total project files in the directory: 503
      [-] Total files to be scanned: 272
      [-] File Extentions Identified: ['.java']
】

**Comment by coffeeandsecurity** [comment]
Yes. It is already in my TODO list. However, as it is not the most important item to track and requires several exclusions to consider such as comments, spaces, CRLF, etc, it is planned to be included at a much later stage.

**Comment by coffeeandsecurity** [comment]
Yes, it is already on my TODO list. However, since it is not the most important item to track and requires several exclusions to consider, such as comments, whitespace, CRLF, and other non-essential elements, it is planned for a much later stage.

I am keeping this issue open for now and will close it once I have included it.

**Comment by coffeeandsecurity** [comment]
With the latest release v0.31, you can now count lines of code using --loc option. 

Hope it helps. Marking this issue as closed.
